src/builtins/Transformers.jl

```diff
@@ -626,27 +636,36 @@ end
 ## ONE HOT ENCODING
 
 """
-    OneHotEncoder(; features=Symbol[], drop_last=false, ordered_factor=true)
+    OneHotEncoder(; features=Symbol[],
+                    ignore=false,
+                    ordered_factor=true,
+                    drop_last=false)
+
+Unsupervised model for one-hot encoding the `Finite` features of some
+table. If `features` is unspecified all features with `Finite` element
+scitype are encoded. Otherwise, encoding is applied to all `Finite`
+features named in `features` (`ignore=false`) or all `Finite` features
+not named in features (`ignore=true`).
 
-Unsupervised model for one-hot encoding all features of `Finite`
-scitype, within some table. If `ordered_factor=false` then
-only `Multiclass` features are considered. The features encoded are
-further restricted to those in `features`, when specified and
-non-empty.
+If `ordered_factor=false` then the above holds with `Finite` replaced
+with `Multiclass`, ie `OrderedFactor` features are not transformed.
 
-If `drop_last` is true, the column for the last level of each
-categorical feature is dropped. New data to be transformed may lack
-features present in the fit data, but no new features can be present.
+Specify `drop_last=true` if the column for the last level of each
+categorical feature is to be dropped.
 
-*Warning:* This transformer assumes that the elements of a categorical
- feature in new data to be transformed point to the same
- CategoricalPool object encountered during the fit.
+New data to be transformed may lack features present in the fit data,
+but no *new* features can be present.
+
+*Warning:* This transformer assumes that the `Multiclass` or
+ `OrderedFactor` features in new data to be transformed have the same
+ underlying `CategoricalVector` encodings as those during the fit.
```

Comment:
An example here would be super helpful

Reply:
This warning is a bit out of date and I have updated it. 

I have also added examples for the OneHotEncoder and ContinuousEncoder